ecs-cli/modules/cli/compose/factory/factory.go

```diff
@@ -59,8 +59,7 @@ func (projectFactory projectFactory) Create(cliContext *cli.Context, isService b
 // populateContext sets the required CLI arguments to the context
 func (projectFactory projectFactory) populateContext(ecsContext *context.Context, cliContext *cli.Context) error {
 	// populate CLI context
-	// TODO: Support multiple compose files
-	ecsContext.ComposeFiles = []string{cliContext.GlobalString(command.ComposeFileNameFlag)}
+	libcomposecommand.Populate(&ecsContext.Context, cliContext)
```

Comment:
Can you add comments here on what this function is doing? and also mention the the default file names that we're reading from.